### PR TITLE
refactor: migrate to circle ci

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: 'circleci/node:latest'
+    steps:
+      - checkout
+      - run:
+          name: install
+          command: npm install
```
Comment (yehjxraymond) [inline .circleci/config.yml]: Wanna cache this? https://circleci.com/docs/2.0/caching/#npm-node

File: README.md [hunk 2]
```diff
@@ -48,4 +48,4 @@ verify(document).then(console.log);
 
 ## License
 
-[Apache v2.0](https://www.apache.org/licenses/LICENSE-2.0)
\ No newline at end of file
+[Apache v2.0](https://www.apache.org/licenses/LICENSE-2.0)
```
Comment (yehjxraymond) [inline README.md]: Our license are all technically GPL now

Comment (Nebulis) [reply README.md]: every project ?

**Pull request merged**